### PR TITLE
docs(pagination): update documentation for Pagination control

### LINKED ISSUE #304: Pagination - Documentation Update
Update component code to work with Dgeni doc generation.  See the [project overview](https://github.com/HealthCatalyst/Fabric.Cashmere/projects) for more details.

### PR DESCRIPTION
**Pull request by joeskeen**
As per request from @andrew-frueh , I have revisited the Pagination control documentation and formatted it to work better with the new documentation system.

Closes #304

### CODE REVIEW

**Comment by jtrujill** [comment]
Before this gets merged, we'll need to edit the  comment so it references the issue #304. @joeskeen if you don't want to do that, I can do that for you.

**Comment by joeskeen** [comment]
@jtrujill Which comment do you mean?  The commit comment, or the PR title?

If you want to go ahead and make the change, that's fine with me.

File: projects/cashmere/src/lib/pagination/pagination.component.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ import {Component, EventEmitter, Input, OnChanges, Output, SimpleChanges} from '
 export class PaginationComponent implements OnChanges {
```
[the remark this answers is not in the document]
Comment (joeskeen) [reply projects/cashmere/src/lib/pagination/pagination.component.ts]: @andrew-frueh done.

**Comment by jtrujill** [comment]
@joeskeen I'll go ahead and make that change for you once I see this passes and I'll get that merged if that's good

**Pull request closed without merging**